examples/simple_trixi_controller.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+
+#include <trixi.h>
+
+int main ( int argc, char *argv[] ) {
+
+    // Initialize Trixi
+    trixi_initialize();
+
+    // An "elixir" file is required to configure the Trixi simulation
+    const char * elixir = "../../LibTrixi.jl/examples/libelixir_demo.jl";
+
+    // Setup the Trixi simulation
+    // We get a handle to use subsequently
+    int handle = trixi_setup_simulation( elixir );
+
+    // Get time step length
+    printf("Current time step length: %f\n", trixi_calculate_dt(handle));
+
+    // Main loop
+    while ( trixi_is_finished( handle ) == 0 ) {
```

Comment:
```suggestion
    while ( !trixi_is_finished( handle ) ) {
```
Since we are in C world here, we can just use the canonical "0 is equal to false". At least IMHO this is more descriptive